src/compliment/context.clj

```diff
@@ -43,6 +43,11 @@
                (str/replace "\\}" "(char 125)")
                (str/replace "{" "(compliment-hashmap ")
                (str/replace "}" ")")
+
+               ;; Basic enhance reader for forms with reader conditionals. Include the meta
+               ;; for sources that are sensitive to this.
+               (str/replace "#?" "^:reader-conditional? ")
+
```

Comment:
`{:read-cond :allow}` didn't have the behavior I wanted... It inevitably read the `:clj` branch of the conditional. I write a bunch of CLJC with specific :cljs branches, but the :cljs branches I'd like to still use completions for haha.

Reply:
I'd love to see a couple tests for this.